dev-cpp/notcurses/notcurses-2.2.2.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS="cmake"
+inherit cmake-multilib
+
+DESCRIPTION="Blingful TUIs and character graphics"
+HOMEPAGE="https://notcurses.com"
+SRC_URI="https://github.com/dankamongmen/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+DEPEND="dev-libs/libunistring
+	media-video/ffmpeg
+	>=sys-libs/readline-8.0"
+RDEPEND="${DEPEND}"
+BDEPEND="doc? ( app-text/pandoc )"
```

Comment:
As you are the upstream, you'd do a huge favor for everyone by shipping pre-generated doc files, if possible. I see `find_package(PkgConfig REQUIRED)` so `BDEPEND` on `virtual/pkgconfig` too.

Reply:
I generate documentation packs with each release, e.g.

https://github.com/dankamongmen/notcurses/releases/download/v2.2.3/notcurses-doc-2.2.3.tar.gz

could you suggest how this tarball can be best integrated with the package overall? Thanks!

Reply:
We can add it into SRC_URI! It accepts multiple (space-separated) elements.

Reply:
cool, i've done that, but then...where do i want to "manually" install these from?

Reply:
also, i see i can make `SOURCE_URI` components dependent upon use variables. ought i make these dependent on `doc`? i.e. is `doc` about the cost of building documentation, or the cost of installing it?

Reply:
I'd install them unconditionally. It's 1 MB file. 

Here it's about the dependencies required to *build* those 1 MB worth of files, installing *pandoc* on my system requires 135 new packages to be installed.

Reply:
Got it. We're unconditionally downloading the file. I'll go ahead and extract it in ... i guess i unpack it in the prepare step, and install in the install step?

Reply:
I have made download of the doc package conditional on doc using the syntax from https://devmanual.gentoo.org/ebuild-writing/variables/. I'm going to conditionally install them from the `src_install` step using the `doman` function. I need to test this with `doc` set, but I think we're good. Thanks for the feedback!